test/functional/test_framework/test_node.py

```diff
@@ -136,3 +136,7 @@ def node_encrypt_wallet(self, passphrase):
             time.sleep(0.1)
         self.rpc = None
         self.rpc_connected = False
+
+    def cli(self, args):
+        """Start the cli."""
+        return subprocess.Popen(['bitcoin-cli'] + args, stderr=self.stderr, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
```

Comment:
We need a way to override what bitcoin-cli to use, akin to `BITCOIND` environment variable.

Which reminds me, didn't we have another PR to add bitcoin-cli testing support at some point?